lib/json.cpp

```diff
@@ -192,8 +192,11 @@ JsonObject* JsonObject::emplace(cstring label, IJson* value) {
     if (label.isNullOrEmpty())
         throw std::logic_error("Empty label");
     auto j = get(label);
-    if (j != nullptr)
-        throw std::logic_error(cstring("Duplicate label in json object ") + label.c_str());
+    if (j != nullptr) {
+      cstring s = value->toString();
+      throw std::logic_error(cstring("Duplicate label in json object ")
```

Comment:
this error seems now a bit incorrect; perhaps it should say: "attempt to add to json object a *value* for a label *label* which already exists.